devices/collect/dac/v3.yaml

```diff
@@ -0,0 +1,80 @@
+_include:
+  - array.yaml
+
+_array:
+  SHSR[12]:
+    description: DAC channel%s sample and hold sample time register
+    _strip_end: "1"
+  "?~STR[12]":
+    _strip_end: "1"
+
+CR:
+  _array:
+    EN[12]:
+      description: DAC channel%s enable
+    TEN[12]:
+      description: DAC channel%s trigger enable
+    WAVE[12]:
+      description: DAC channel%s noise/triangle wave generation enable
+    MAMP[12]:
+      description: DAC channel%s mask/amplitude selector
+    DMAEN[12]:
+      description: DAC channel%s DMA enable
+    DMAUDRIE[12]:
+      description: DAC channel%s DMA Underrun Interrupt enable
+    CEN[12]:
+      description: DAC channel%s calibration enable
+
+"SWTRGR,SWTRIGR":
+  _array:
+    SWTRIG[12]:
```

Comment:
Looking at DAC_SWTRGR in [RM0440](https://www.st.com/resource/en/reference_manual/rm0440-stm32g4-series-advanced-armbased-32bit-mcus-stmicroelectronics.pdf), should this also be done for SWTRIG**B**1 and SWTRIG**B**2?